modules/peers.js

```diff
@@ -429,9 +430,11 @@ __private.dbSave = function(cb) {
 	}
 
 	// Wrap sql queries in transaction and execute
-	return library.db
-		.tx('modules:peers:dbSave', t =>
-			t.peers.clear().then(() => t.peers.insert(peers))
+	return library.storage.adapter
+		.transaction('modules:peers:dbSave', t =>
```

Comment:
Better to write 

`library.storage.entities.Peer.begin`, instead of directly calling adapter method.

Reply:
Done.